tests/kubernetes/graph/test_running_graph_checks.py

```diff
@@ -0,0 +1,19 @@
+import itertools
+import os
+import unittest
+
+from checkov.kubernetes.runner import Runner
+
+TEST_DIRNAME = os.path.dirname(os.path.realpath(__file__))
+
+
+class TestRunningGraphChecks(unittest.TestCase):
+    def test_runner(self):
+        root_dir = os.path.realpath(os.path.join(TEST_DIRNAME, "../runner/resources"))
+        report = Runner().run(root_dir)
+        assert any(
+            check.check_id == "CKV2_K8S_21" for check in itertools.chain(report.failed_checks, report.passed_checks))
```

Comment:
@YaaraVerner please also check the logic isn't reversed by verifying counts of passed and failed 🙏